src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerFieldsTest.php

```diff
@@ -500,8 +470,7 @@ public function testGetTreeActionSingleWebspaceNodes()
 
     public function testGetTreeActionAllWebspaceNodes()
     {
-        $this->initPhpcr();
-        $this->createWebspaceRoot('test_io');
+        $this->createHomeDocument('test_io', ['de']);
```

Comment:
It suddenly comes from SuluTestCase. It creates the webspace root (home document) using the document manager (rather than directly using PHPCR)